plugins/node/opentelemetry-instrumentation-aws-sdk/test/sqs.test.ts

```diff
@@ -36,12 +36,19 @@ import { ReadableSpan } from '@opentelemetry/sdk-trace-base';
 import { mockV2AwsSend } from './testing-utils';
 import { Message } from 'aws-sdk/clients/sqs';
 import * as expect from 'expect';
+import * as sinon from 'sinon';
+import * as messageAttributes from '../src/services/MessageAttributes';
 
 const responseMockSuccess = {
   requestId: '0000000000000',
   error: null,
 };
 
+const extractContextSpy = sinon.spy(
```

Comment:
Creating a spy on this scope effects all tests. You could move it into the relevant test/testsuite and instead of calling `resetHistory()` you could create a new spy for each test and use `sinon.restore()` after each test.